AppCenter/AppCenter/Internals/Ingestion/MSOneCollectorIngestion.m

```diff
@@ -94,6 +97,38 @@ - (NSURLRequest *)createRequest:(NSObject *)data
                     stringWithFormat:@"%lld",
                                      (long long)[MSUtility nowInMilliseconds]]
          forKey:kMSOneCollectorUploadTimeKey];
+
+  // Gather tokens from logs.
+  NSMutableString *ticketKeyString = [NSMutableString new];
+  for (id<MSLog> log in container.logs) {
+    MSCommonSchemaLog *csLog = (MSCommonSchemaLog *)log;
+    if (csLog.ext.protocolExt) {
+      NSArray<NSString *> *ticketKeys = [[[csLog ext] protocolExt] ticketKeys];
+      for (NSString *ticketKey in ticketKeys) {
+        NSString *token = [[MSTicketCache sharedInstance] ticketFor:ticketKey];
```

Comment:
Not a blocker but we might be confused about what those tokens are, token variable name might be a bit more explicit like authenticationToken or something.